PaulKariukiRimiru/Team-timer
# Fix reset timer bug

#### What does this PR do?
Fixes the reset-timer bug

#### How should this be manually tested?
- Fetch the branch locally
- Run `npm install` to install necessary dependencies.
- Run `npm start` to start the application. Select a team-mate and wait for timer to reach zero.
- Select another team-mate. Timer should begin from 60 seconds again.
